CONTRIBUTING.md

```diff
@@ -105,11 +105,9 @@ Code all lives in the [dotnet/samples](https://github.com/dotnet/samples) reposi
 
 - *snippets* contains the code snippets used throughout the documentation. Inside the *snippets* folder, the folder names identify the language of the code snippet.
```

Comment:
it seems that we were also adopting the path `https://github.com/dotnet/samples/tree/master/snippets/<language>/api/<namespace>/<apiname>` for api reference snippets as you can see at https://github.com/dotnet/samples/tree/master/snippets/csharp/api

also any suggestions for folder name when we're creating cross-plat samples that will go under the standard folder? They will be .NET Core SDK-style projects.